packages/react-components/styles/colors.v2.tsx

```diff
@@ -0,0 +1,39 @@
+// Designer Created Figma Colors
+export enum colors {
+  greenFaint = '#97DFC1', // green disabled
+  greenBrand = '#42D689',
+  greenUI = '#1AB775',
+  goldFaint = '#E3C376', // gold disabled
+  goldBrand = '#FBCC5C',
+  goldUI = '#EEB93C',
+  goldDark = '#9C6E00',
+  beige = '#F5F4F0',
+  brownFaint = '#FFF9EE',
+  warning = '#EA6042',
+  dark = '#2E3338',
+  gray5 = '#81868B',
+  gray4 = '#9CA4A9',
+  gray3 = '#B4B9BD',
+  gray2 = '#EDEEEF',
+  gray1 = '#F8F9F9',
+  light = '#FFFFFF',
+}
+
+// deprecated transition map of old names new colors
```

Comment:
Just an idea here for the future - add `getter` and show a warning in dev mode, that newer color names should be used instead.